GoogleDataLogger/GoogleDataLogger/Classes/Public/GDLLogScorer.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+@class GDLLogEvent;
+
+/** This protocol defines the common interface of a log scorer. Log scorers are stateful objects
+ *  that score logs upon insertion into storage and remain prepared to return a set of log filenames
+ *to the storage system
```

Comment:
Nit: spacing, ending punctuation.

Reply:
Corrected. Seems like a possible bug in style.sh